README.md

```diff
@@ -0,0 +1,125 @@
+[![CircleCI](https://circleci.com/gh/apostrophecms/sync-content/tree/master.svg?style=svg)](https://circleci.com/gh/apostrophecms/sync-content/tree/master)
+[![Chat on Discord](https://img.shields.io/discord/517772094482677790.svg)](https://chat.apostrophecms.org)
+
+<p align="center">
+  <a href="https://github.com/apostrophecms/apostrophe">
+    <!-- TODO:  -->
+    <img src="https://raw.githubusercontent.com/apostrophecms/apostrophe/main/logo.svg" alt="ApostropheCMS logo" width="80" height="80">
+  </a>
+
+  <h1 align="center">Sync Content for ApostropheCMS</h1>
+</p>
+
+The Sync Content module allows syncing ApostropheCMS site content between different server environments, without the need for direct access to remote databases, directories, S3 buckets, etc.
+
+**Status:** ⚠️ In use, but still an alpha release. Unimplemented features are noted below with a 🚧. Completed features are noted with a ✅.
```

Comment:
The symbol key isn't clearly a key. Especially the first one. This should be some sort of unordered list.

Reply:
I just realized the warning sign isn't part of the key. I guess it's fine as is.